docker/common.sh

```diff
@@ -1,8 +1,7 @@
 #!/bin/bash
 
 function run_deployment_tests() {
-  export XRT_DEVICE_MAP="CPU:0;/job:localservice/replica:0/task:0/device:XLA_CPU:0"
-  export XRT_WORKERS="localservice:0;grpc://localhost:40934"
+  export PJRT_DEVICE=TPU
```

Comment:
maybe `PJRT_DEVICE=TPU`? the above command is to configure XRT CPU

Reply:
Thanks for pointing, test in https://github.com/pytorch/xla/pull/5220

Reply:
#5220's change has been saved in https://github.com/pytorch/xla/pull/5222 too, since https://github.com/pytorch/xla/pull/5222 has passed all tests, will apply this change in #5220

Reply:
Thanks, updated